pycm/pycm_obj.py

```diff
@@ -866,3 +866,25 @@ def to_array(self, normalized=False, one_vs_all=False, class_name=None):
             row = [table[key][i] for i in classes]
             array.append(row)
         return numpy.array(array)
+
+    def combine(self, other):
+        """
+        Return the combine of self Confusion matrix and other one.
+
+        :param other: Other matrix that is going to combine.
+        :type other: Confusion matrix
+        :return: combine of two matrices as Confusion matrix
+        """
+        result_matrix = {}
```

Comment:
Please move this section (`878` to `889`) to a function in `pycm_util.py`.
Call it `matrix_combine`